.ci/Dockerfile.fedora33

```diff
@@ -0,0 +1,39 @@
+FROM fedora:33
+
+RUN dnf install -y \
```

Comment:
do you really need all these packages to check formatting?

Reply:
This Fedora docker image can be also used for static code analysis with clang analyzer if we'd like it (otherwise we'll remove some unneeded packages).